confiture-rest-api/src/auth/auth.controller.ts

```diff
@@ -318,6 +319,20 @@ export class AuthController {
     }
   }
 
+  @Post('account/cancel-email-update')
+  async cancelEmailUpdate(@Body() body: CancelEmailUpdateDto) {
+    try {
```

Comment:
Utiliser le compte connecté plutôt que de passer l'email dans le body.

```suggestion
  @Post('account/cancel-email-update')
  @AuthRequired()
  async cancelEmailUpdate(@User() user: AuthenticationJwtPayload) {
    await this.auth.cancelEmailUpdate(user.email);
```